routers/actionsRouter.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+const router = express.Router();
+const Actions = require("../data/helpers/actionModel");
+
+
+
+router.get('/', (req, res) => {
+  Actions.get()
+    .then(actns => {
+      console.log("Actions router, .get actns", actns);
+      res.status(200).json(actns);
+    })
+})
+
+
+router.post('/', (req, res) => {
+  const projId = req.body.project_id;
+  if (!projId) { return status(400).json({ errorMessage: "There's no post id for this action" }) }
+  Actions.insert(req.body)
```

Comment:
Don't forget the extra checks to ensure every action has a `description` and a `name` property! 😮